js/w3c/templates/webspecs-headers.html

```diff
@@ -0,0 +1,20 @@
+<header>
+  <a href="http://specs.webplatform.org/" id="logo">
+    <img alt="Web Platform Specs logo" height="50" src="https://specs.webplatform.org/assets/img/logo.svg" width="50"><br>
```

Comment:
Yeah, the `span` was a leftover from other things. The `br` is needed though, this is poetry.